index-handlers/src/main/java/no/sikt/nva/nvi/index/xlsx/Excel.java

```diff
@@ -0,0 +1,69 @@
+package no.sikt.nva.nvi.index.xlsx;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.Base64;
+import java.util.Base64.Encoder;
+import java.util.List;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+
+public record Excel(Workbook workbook) {
```

Comment:
Could use a better name.

Reply:
Something that describes what the record stores, or does.

Reply:
@sondrev Any suggestions? `WorkbookWrapper`?